### DISCUSSION TITLE
CodeFec部署问题

### DISCUSSION THREAD

**Discussion by jackberry** [zhuchunshu/CodeFec-Htshop#5] [Q&A]
我按照https://bbs.vpsdie.com/wiki/UseCodeFec这个教程在宝塔面板下安装好环境，运行一键安装CodeFec时报下面的错误

2021-06-16 16:55:31 (31.5 MB/s) - ‘cn-install.sh.2’ saved [670]

PHP Warning:  require(/root/CodeFec/CodeFec/vendor/autoload.php): failed to open stream: No such file or directory in /root/CodeFec/CodeFec/artisan on line 18
PHP Fatal error:  require(): Failed opening required '/root/CodeFec/CodeFec/vendor/autoload.php' (include_path='.:/usr/share/php') in /root/CodeFec/CodeFec/artisan on line 18

**Comment by zhuchunshu** [comment]
在CodeFec根目录下执行: composer install
